apis/r/tools/build_libtiledbsoma.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+## This script is invoked from the tarball-getting script `get_tarballs.R` which
+## itself is called from `configure`.
+
+cwd=`pwd`
+
+## This helps a little
```

Comment:
Can you clarify what this helps with?

Reply:
The following line normalizes the path (unique inside the tarball it came from) to a non-versioned one as then used in the remainder.